lib/bourbon.rb

```diff
@@ -1,17 +1,9 @@
-# CodeKit needs relative paths
```

Comment:
Oh, I dumped all the Ruby content of Bourbon a long time ago. The only real reason the gem existed was to copy the Bourbon stylesheet assets into a specified location and to return the version of Bourbon. CodeKit took over those tasks itself so all that's bundled into CodeKit are the stylesheet assets from Bourbon.